### PR TITLE
Error if include_homepage_link is not in $settings

### PR DESCRIPTION
**Pull request by jjborie**
I am targetting this branch, because…

Closes #
Fixes #
## Changelog

``` markdown
### Added
- Added some `Class::newMethod` to do great stuff

### Changed

### Deprecated

### Removed

### Fixed

### Security
```
## To do

- [ ] Update the tests
- [ ] Update the documentation
- [ ] Add an upgrade note
## Subject

I don't use breadcrump and I don't have include_homepage_link. If it's the case you have this error:

Fatal error: Uncaught exception 'Symfony\Component\Debug\Exception\ContextErrorException' with message 'Notice: Undefined index: include_homepage_link' in /var/www/leaders/vendor/sonata-project/seo-bundle/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php:129

### CODE REVIEW

**Comment by core23** [comment]
Please update your PR description, so all dummy text is removed and replaced by some real information.

File: Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php [hunk 1]
```diff
@@ -127,8 +127,10 @@ protected function getRootMenu(BlockContextInterface $blockContext)
             $menu->setCurrent($settings['current_uri']);
         }
 
-        if ($settings['include_homepage_link']) {
```
Comment (OskarStark) [inline Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php]: to keep the diff small i would use

```php
if (isset($settings['include_homepage_link']) && $settings[include_homepage_link'])
```

**Comment by core23** [comment]
Please rebase @jjborie

**Comment by core23** [comment]
Please update the PR template and rebase @jjborie

**Comment by greg0ire** [comment]
ping @Soullivaneuh stuck build

File: composer.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
         "symfony/framework-bundle": "^2.3 || ^3.0",
         "symfony/options-resolver": "^2.3 || ^3.0",
         "sonata-project/exporter": "^1.2.2",
-        "sonata-project/block-bundle": "^3.1.1",
+        "sonata-project/block-bundle": "3.0.1",
```
Comment (OskarStark) [inline composer.json]: why?

**Comment by OskarStark** [comment]
We're still using Scrutinizer in this Repo ❓

File: composer.json (hunk 2 above)
Comment (core23) [inline composer.json]: Please revert @jjborie

**Comment by core23** [comment]
Ping @jjborie

**Comment by core23** [comment]
Closing this, 'cause no response from @jjborie 

Also, we don't need this fix, because `include_homepage_link` exists in the default configuration: https://github.com/sonata-project/SonataSeoBundle/blob/2.x/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php#L83

**Pull request closed without merging**